ci-operator/step-registry/openshift-qe/installer/bm/day2/label/openshift-qe-installer-bm-day2-label-commands.sh

```diff
@@ -9,13 +9,17 @@ if [ ${BAREMETAL} == "true" ]; then
   SSH_ARGS="-i /bm/jh_priv_ssh_key -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
   bastion="$(cat /bm/address)"
   # Copy over the kubeconfig
-  ssh ${SSH_ARGS} root@$bastion "cat ${KUBECONFIG_PATH}" > /tmp/kubeconfig
+  if [ ! -f "${SHARED_DIR}/kubeconfig" ]; then
```

Comment:
@vishnuchalla are you confident that this change won't impact the daily jobs that consume the label step?

Reply:
Yes we are sure on this. Check `bm-deploy` step, which copies kubeconfig to shared directory at the end.